src/TriggerBinding/SqlTriggerBindingProvider.cs

```diff
@@ -57,6 +57,10 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             ParameterInfo parameter = context.Parameter;
             SqlTriggerAttribute attribute = parameter.GetCustomAttribute<SqlTriggerAttribute>(inherit: false);
 
+            // During application startup, the WebJobs SDK calls 'TryCreateAsync' method of all registered trigger
+            // binding providers in sequence for each parameter in the user function. A provider that finds the
+            // parameter-attribute that it can handle returns the binding object. Rest of the providers are supposed to
+            // return null. This binding object later gets used for binding before every function invocation.
```

Comment:
I'd suggest also updating it to directly return null instead of `default(ITriggerBinding)` since that's the value we actually want.

Reply:
This was done to prevent explicitly passing the type-parameter `ITriggerBinding` to call the generic `Task.FromResult` method. Calling this way keeps lines 66 and 88 similar.

Reply:
Line 68? I'm not seeing anything in that line that looks similar. 

And I guess I just don't see the point in doing `default(ITriggerBinding)`. Your comment (and the functionality expected) is that null is returned if the provider can't handle the parameter. So we should just return null. Keep it simple and straightforward.

Reply:
It's line 84 in old revision and line 88 in new revision.